index.html

```diff
@@ -19,6 +19,18 @@
             <img class="social_media_icons" src="icons/Union.png" alt="">
         </nav>
     </header>
+    <!--Desktop View-->
+    <header class="desktop_header">
+        <!--header section-->
+        <nav class="desktop_navigation_bar">
+            <p><span>My Logo</span></p>
+            <ul class="nav_item">
+                <li class="nav_item-li1"><a>Portoflio</a></li>
+                <li class="nav_item-li2"><a>About</a></li>
+                <li class="nav_item-li3"><a>Contact</a></li>
+            </ul>
```

Comment:
- [ ] Kindly add the feature that will allow the user to use the nav-links to navigate through the page by being redirected to a corresponding section after a click 

```bash
....
<li class="nav_item-li1"><a href="#section-id">Portoflio</a></li>

....

<section id= "section-id">
....
```